core/consensus/babe/Cargo.toml

```diff
@@ -27,6 +27,7 @@ log = "0.4.6"
 schnorrkel = "0.1.1"
 rand = "0.6.5"
 merlin = "1.0.3"
+curve25519-dalek = { version = "1.2.1", features = ["u64_backend"] }
```

Comment:
why add this dependency?

Reply:
I was trying to fix a compile-time error.